### PR TITLE
Upgrade node-version to 16 (was 14) in GitHub Actions workflows

### PR DESCRIPTION
**Pull request by dconeybe**
Upgrade `node-version` to 16 (was 14) in GitHub Actions workflows (`desktop.yml` and `integration_tests.yml`).

Node version 16 is the "long term support" version.

This may also fix intermittent failures of `npm install -g firebase-tools` with the error "npm ERR! cb() never called!"

### CODE REVIEW

File: .github/workflows/desktop.yml [hunk 1]
```diff
@@ -357,7 +357,7 @@ jobs:
       - name: Set up Node (14)
```
Comment (sunmou99) [inline .github/workflows/desktop.yml:357]: Nit: Set up Node (16)

Comment (dconeybe) [reply .github/workflows/desktop.yml]: Done.

**Comment by jonsimantov** [comment]
Is this incompatible with the retry PR? (I'm merging that one - I figure it's still best to retry any network-related thing if possible).

**Comment by dconeybe** [comment]
> Is this incompatible with the retry PR? (I'm merging that one - I figure it's still best to retry any network-related thing if possible).

No. They are compatible. I agree, that the retry is useful regardless of which node version is being installed.

**Review by sunmou99 — APPROVED** [review]

**Pull request merged**